### PR TITLE
fix [118] verify mapping between api object and what is display in forms

### PR DESCRIPTION
**Pull request by Donatien26**
Signed-off-by: Donatien Eneman <donatien.eneman@insee.fr>

### CODE REVIEW

**Comment by Donatien26** [comment]
close #118

File: src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigOrganization.tsx [hunk 1]
```diff
@@ -60,16 +60,39 @@ export default [
...
+		tag: 'advanced',
+	},
+	{
+		name: 'UserStorage',
+		path: 'metadatas.userStorage',
+		type: 'string',
+		modifiable: false,
+		tag: 'advanced',
+	},
+	{
```
Comment (CChemin) [inline src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigOrganization.tsx:77]: I think we should not display it, it's linked to ldap + a certain way to use it that we will hopefully drop soon

Comment (Donatien26) [reply src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigOrganization.tsx:77]: It's an advanced field so we don't care no ?

File: src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigUser.tsx [hunk 2]
```diff
@@ -177,7 +200,7 @@ const userDisplayField: field[] = [
 		helpTextTitle: 'Propriétés du contact',
 		helpText:
 			'Ajouter une propriété à saisir en cliquant sur "Ajouter un champ". Supprimer une propriété saisie en vidant le champ ou en cliquant sur la croix rouge à droite du champ. Aucune limite d\'ajout.',
-		path: 'roles',
+		path: 'attributes.properties',
 		type: 'list',
 		tag: 'properties',
 		modifiable: true,
```
Comment (CChemin) [inline src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigUser.tsx:206]: not implemented so modifiable: false ?

File: src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigOrganization.tsx (hunk 1 above)
Comment (CChemin) [reply src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigOrganization.tsx:77]: ok

File: src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigUser.tsx (hunk 2 above)
Comment (Donatien26) [reply src/hooks/realm/useRealmConfig/fieldToDisplay/FieldToDisplayConfigUser.tsx:206]: yep i change it

**Review by CChemin — APPROVED** [review]

**Pull request merged**